Cargo.lock

```diff
@@ -1593,6 +1593,7 @@ version = "0.0.0"
 dependencies = [
  "chrono",
  "fuel-core",
+ "fuel-crypto",
```

Comment:
Why is the lockfile being updated? If it wasn't updated in a previous PR, can you separate this into a different PR that can be reviewed independently?

Reply:
because I needed to add fuel-crypto as a dependency to the integration tests in order to write the test needed to repro the issue.

Reply:
Oh I see now, I missed the manifest file change.